### PR TITLE
fix: CVE issues

### PR DESCRIPTION
**Pull request by c3y1huang**
https://github.com/longhorn/longhorn/issues/6916
https://github.com/longhorn/longhorn/issues/6922
https://github.com/longhorn/security/issues/16

### CODE REVIEW

File: deploy/longhorn-images.txt [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-longhornio/csi-resizer:v1.7.0
-longhornio/csi-snapshotter:v6.2.1
-longhornio/csi-node-driver-registrar:v2.7.0
-longhornio/livenessprobe:v2.9.0
+longhornio/csi-attacher:v4.4.0
+longhornio/csi-provisioner:v3.6.0
+longhornio/csi-resizer:v1.9.0
+longhornio/csi-snapshotter:v6.3.0
+longhornio/csi-node-driver-registrar:v2.9.0
+longhornio/livenessprobe:v2.11.0
```
Comment (innobead) [inline deploy/longhorn-images.txt:6]: Have we run e2e testing to ensure these changes will not cause regression? Have we checked if any breaking changes compared to the current used versions?

Comment (c3y1huang) [reply deploy/longhorn-images.txt:6]: We've run the [coretest](https://ci.longhorn.io/job/private/job/longhorn-tests-regression/5048/) with the current version longhorn components and also with the [CVE fixed components](https://github.com/longhorn/security/issues/16#issuecomment-1770216981).

**Comment by c3y1huang** [comment]
**BLOCKED**
This PR is waiting for support-bundle-kit fix. Currently experiencing S390X drown worker issuer. https://suse.slack.com/archives/C02CN30DWUF/p1698022440204029

**Comment by c3y1huang** [comment]
> **BLOCKED:** This PR is waiting for support-bundle-kit fix. Currently experiencing S390X drown worker issue. https://suse.slack.com/archives/C02CN30DWUF/p1698022440204029 [harvester/harvester#4642](https://github.com/harvester/harvester/issues/4642)

**UNBLOCKED:** https://drone-publish.rancher.io/rancher/support-bundle-kit/239

**Comment by c3y1huang** [comment]
Waiting for support-bundle-kit image to mirror to longhornio. 
https://github.com/longhorn/longhorn/issues/6922#issuecomment-1784677004

**Comment by c3y1huang** [comment]
ref https://github.com/longhorn/longhorn/issues/7001

**Comment by c3y1huang** [comment]
> **BLOCKED:** [#6922 (comment)](https://github.com/longhorn/longhorn/issues/6922#issuecomment-1784677004)

**UNBLOCKED**

**Review by innobead — APPROVED** [review]

**Comment by c3y1huang** [comment]
@mergify backport v1.5.x

**Comment by innobead** [comment]
@c3y1huang will we backport this to v1.4.x?

**Comment by c3y1huang** [comment]
> @c3y1huang will we backport this to v1.4.x?

We won't be updating the CSI sidecar since we're not upgrading the Kubernetes version in v1.4.x.
https://github.com/longhorn/longhorn/issues/5672#issuecomment-1502540522

**Pull request merged**